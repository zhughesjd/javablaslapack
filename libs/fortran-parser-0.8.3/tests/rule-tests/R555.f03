

! Testing equivalence set list. Very similar to test for equivalence objects.
equivalence (a,b(3)), (x,y(2),z(4))
end
