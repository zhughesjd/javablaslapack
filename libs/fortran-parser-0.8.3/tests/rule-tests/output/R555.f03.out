1101:begin
555:list-begin
613 a false false
612 1
609 false
556:list-begin
619:list-begin
406 3 null
619 true false false true
619:list 1
613 b true false
612 1
609 false
556:list 1
613 x false false
612 1
609 false
556:list-begin
619:list-begin
406 2 null
619 true false false true
619:list 1
613 y true false
612 1
609 false
619:list-begin
406 4 null
619 true false false true
619:list 1
613 z true false
612 1
609 false
556:list 2
555:list 2
1103
1101 false false false
